pkg/runner/run_context.go

```diff
@@ -120,7 +120,7 @@ func (rc *RunContext) startJobContainer() common.Executor {
 			rc.stopJobContainer(),
 			rc.JobContainer.Create(),
 			rc.JobContainer.Start(false),
-			rc.JobContainer.CopyDir(copyToPath, rc.Config.Workdir+string(filepath.Separator)+".").IfBool(copyWorkspace),
+			rc.JobContainer.CopyDir(copyToPath, rc.Config.Workdir+string(filepath.Separator)+".", rc.Config.AllowGitignore).IfBool(copyWorkspace),
```

Comment:
I don't think this is working as intended. The `--allow-gitignore` flag is default to `false` so the default behavior is now to "allow git ignore" entries. I think the current default behavior should be preserved.

Reply:
I think I got confused somewhere around looking through code and PR. Will correct that this weekend (or earlier if I'll get some time)

Reply:
Fixed